common/src/View/UVViewHelper.cpp

```diff
@@ -205,25 +205,23 @@ namespace TrenchBroom {
 
             const auto toTex = m_face->toTexCoordSystemMatrix(vm::vec2f::zero, vm::vec2f::one, true);
             const auto transformedVertices = toTex * m_face->vertexPositions();
-            const auto bounds = vm::bbox3::mergeAll(std::begin(transformedVertices), std::end(transformedVertices));
-
-            const vm::vec3 vertices[] = {
-                bounds.corner(vm::bbox3::Corner::min, vm::bbox3::Corner::min, vm::bbox3::Corner::min),
-                bounds.corner(vm::bbox3::Corner::min, vm::bbox3::Corner::max, vm::bbox3::Corner::min),
-                bounds.corner(vm::bbox3::Corner::max, vm::bbox3::Corner::max, vm::bbox3::Corner::min),
-                bounds.corner(vm::bbox3::Corner::max, vm::bbox3::Corner::min, vm::bbox3::Corner::min)
-            };
 
             const auto fromTex = m_face->fromTexCoordSystemMatrix(vm::vec2f::zero, vm::vec2f::one, true);
             const auto toCam = vm::mat4x4(m_camera.viewMatrix());
 
-            for (size_t i = 0; i < 4; ++i) {
-                const auto vertex = toCam * fromTex * vertices[i];
-                if (vertex.x() < 0.0 && vertex.y() < 0.0) {
-                    setOriginInFaceCoords(vm::vec2f(vertices[i]));
-                    break;
+            auto originFace = transformedVertices[0];
```

Comment:
Comment: `// Find a vertex with either the minimum X or Y camera space coordinate`?

Reply:
Done